### PR TITLE
Fixes for STM32MP13 clock driver

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

**Comment by etienne-lms** [comment]
Rebased to get CI tests fixed.

**Review by GseoC — COMMENTED** [review]
With comment addressed:

`Reviewed-by: Gatien Chevallier <gatien.chevallier@foss.st.com>`

File: core/drivers/clk/clk-stm32mp13.c [hunk 1]
```diff
@@ -2116,74 +2116,74 @@ static STM32_TIMER(ck_timg2, &ck_pclk2, 0, RCC_APB2DIVR, RCC_TIMG2PRER);
...
+static STM32_GATE(ck_dmamux2, &ck_axi, 0, GATE_DMAMUX2);
+static STM32_GATE(ck_dma3, &ck_axi, 0, GATE_DMA3);
+static STM32_GATE(ck_tsc, &ck_axi, 0, GATE_TSC);
+static STM32_GATE(ck_aximc, &ck_axi, 0, GATE_AXIMC);
+static STM32_GATE(ck_crc1, &ck_axi, 0, GATE_CRC1);
+static STM32_GATE(ck_eth1tx, &ck_axi, 0, GATE_ETH1TX);
+static STM32_GATE(ck_eth1rx, &ck_axi, 0, GATE_ETH1RX);
+static STM32_GATE(ck_eth2tx, &ck_axi, 0, GATE_ETH2TX);
+static STM32_GATE(ck_eth2rx, &ck_axi, 0, GATE_ETH2RX);
+static STM32_GATE(ck_eth2mac, &ck_axi, 0, GATE_ETH2MAC);
```
Comment (GseoC) [inline core/drivers/clk/clk-stm32mp13.c:2187]: We're missing SPI clocks:
`
static STM32_GATE(ck_spi1, &ck_pclk2, 0, GATE_SPI1);
static STM32_GATE(ck_spi2, &ck_pclk1, 0, GATE_SPI2);
static STM32_GATE(ck_spi3, &ck_pclk1, 0, GATE_SPI3);
static STM32_GATE(ck_spi4, &ck_pclk6, 0, GATE_SPI4);
static STM32_GATE(ck_spi5, &ck_pclk6, 0, GATE_SPI5);
`
and ADC clocks:
`
static STM32_GATE(ck_adc1, &ck_mlahb, 0, GATE_ADC1);
static STM32_GATE(ck_adc2, &ck_mlahb, 0, GATE_ADC2);
`

Comment (etienne-lms) [reply core/drivers/clk/clk-stm32mp13.c:2187]: ok, i'l  create another P-R for them.

**Review by etienne-lms — COMMENTED** [review]
Review tag applied.

**Comment by etienne-lms** [comment]
Review tag applied.
Thanks

File: core/drivers/clk/clk-stm32mp13.c (hunk 1 above)
Comment (etienne-lms) [reply core/drivers/clk/clk-stm32mp13.c:2187]: Addressed by https://github.com/OP-TEE/optee_os/pull/6663.

**Pull request merged**